app/src/renderer/components/network/PageBlock.vue

```diff
@@ -0,0 +1,151 @@
+<template>
+  <tm-page data-title="Block">
+    <template slot="menu-body">
+      <tm-balance />
+      <tool-bar />
+    </template>
+
+    <tm-data-error v-if="!connected || !block" />
+
+    <template v-else>
+      <div class="page-profile__header page-profile__section block">
+        <div class="row">
+          <div class="page-profile__header__info">
+            <div class="page-profile__status-and-title">
+              <h2 class="page-profile__title">
+                Block {{ `#` + num.prettyInt(block.block.header.height) }}
+              </h2>
+              <h3 class="page-profile__subtitle">
+                {{ block.block_meta.block_id.hash }}
+              </h3>
+            </div>
+          </div>
+        </div>
+
+        <div class="row">
+          <dl class="info_dl colored_dl">
+            <dt>Time</dt>
+            <dd>
+              {{
+                moment(block.block.header.time).format("MMM Do YYYY, HH:mm:ss")
```

Comment:
would be nicer to move this function downstairs in the computed ones, so it can be tested OR even nicer to move this in the time helper and have it defined in a single place